corehq/util/datadog/utils.py

```diff
@@ -87,3 +107,8 @@ def get_url_group(url):
         return parts[3] if len(parts) >= 4 else default
 
     return default
+
+
+def notify_datadog(metrics= {}):
```

Comment:
I'd rename this so give more indication of the purpose: `update_datadog_guage_metric`